From 8ea7ff275567f53409b64b69d5a405c78ea93c8c Mon Sep 17 00:00:00 2001
From: Mike Lui <mike.d.lui@gmail.com>
Date: Wed, 7 Mar 2018 16:54:10 -0500
Subject: [PATCH] patch

---
 Makefile.am  | 2 ++
 configure.ac | 4 ++++
 2 files changed, 6 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index fdce3cf9f..6859f9611 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -10,6 +10,8 @@ TOOLS =		memcheck \
 		lackey \
 		none \
 		helgrind \
+		sigrind \
+		gengrind \
 		drd
 
 EXP_TOOLS = 	exp-sgcheck \
diff --git a/configure.ac b/configure.ac
index 4d4521896..624c52200 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4523,6 +4523,10 @@ AC_CONFIG_FILES([
    exp-dhat/tests/Makefile
    shared/Makefile
    solaris/Makefile
+   sigrind/Makefile
+   sigrind/tests/Makefile
+   gengrind/Makefile
+   gengrind/tests/Makefile
 ])
 AC_CONFIG_FILES([coregrind/link_tool_exe_linux],
                 [chmod +x coregrind/link_tool_exe_linux])
-- 
2.16.2

