From 2a62dc36c0e54469e17a5cd73d7c9f019e0b31c3 Mon Sep 17 00:00:00 2001
From: Mike Lui <mike.d.lui@gmail.com>
Date: Wed, 20 Feb 2019 17:15:50 -0500
Subject: [PATCH] patch

---
 Makefile.am  | 2 ++
 configure.ac | 4 ++++
 2 files changed, 6 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 631c845ab..a46d8a74d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -11,6 +11,8 @@ TOOLS =		memcheck \
 		massif \
 		dhat \
 		lackey \
+		sigrind \
+		gengrind \
 		none
 
 EXP_TOOLS = 	exp-sgcheck \
diff --git a/configure.ac b/configure.ac
index b885bd91b..59f37ab65 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4671,6 +4671,10 @@ AC_CONFIG_FILES([
    exp-bbv/tests/arm-linux/Makefile
    shared/Makefile
    solaris/Makefile
+   sigrind/Makefile
+   sigrind/tests/Makefile
+   gengrind/Makefile
+   gengrind/tests/Makefile
 ])
 AC_CONFIG_FILES([coregrind/link_tool_exe_linux],
                 [chmod +x coregrind/link_tool_exe_linux])
-- 
2.20.1

