From bea3479c1fbe1c78a13ac2a4986427580d5a7cfb Mon Sep 17 00:00:00 2001
From: Mike Lui <mike.d.lui@gmail.com>
Date: Wed, 19 Aug 2020 13:27:42 -0400
Subject: [PATCH] Sigrind/Gengrind build

---
 Makefile.am  | 2 ++
 configure.ac | 4 ++++
 2 files changed, 6 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 08db83401..fd5047bca 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,6 +12,8 @@ TOOLS = \
 		massif \
 		dhat \
 		lackey \
+		sigrind \
+		gengrind \
 		none
 
 EXP_TOOLS = \
diff --git a/configure.ac b/configure.ac
index b86e80220..075f64775 100755
--- a/configure.ac
+++ b/configure.ac
@@ -4859,6 +4859,10 @@ AC_CONFIG_FILES([
    exp-bbv/tests/arm-linux/Makefile
    shared/Makefile
    solaris/Makefile
+   sigrind/Makefile
+   sigrind/tests/Makefile
+   gengrind/Makefile
+   gengrind/tests/Makefile
 ])
 AC_CONFIG_FILES([coregrind/link_tool_exe_linux],
                 [chmod +x coregrind/link_tool_exe_linux])
-- 
2.28.0

