From 94412cc5c955953b1c88f231982b1a691dfd75db Mon Sep 17 00:00:00 2001
From: Mike Lui <mike.d.lui@gmail.com>
Date: Sun, 14 Apr 2019 00:03:48 -0400
Subject: [PATCH] patch

---
 include/spdlog/tweakme.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/include/spdlog/tweakme.h b/include/spdlog/tweakme.h
index b3b71e4..fdc7595 100644
--- a/include/spdlog/tweakme.h
+++ b/include/spdlog/tweakme.h
@@ -18,7 +18,7 @@
 // the kernel HZ.
 // Uncomment to use it instead of the regular clock.
 //
-// #define SPDLOG_CLOCK_COARSE
+#define SPDLOG_CLOCK_COARSE
 ///////////////////////////////////////////////////////////////////////////////
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -31,7 +31,7 @@
 //          You must set new pattern(spdlog::set_pattern(..") without any
 //          date/time in it
 //
-// #define SPDLOG_NO_DATETIME
+#define SPDLOG_NO_DATETIME
 ///////////////////////////////////////////////////////////////////////////////
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -41,7 +41,7 @@
 // WARNING: If the log pattern contains thread id (i.e, %t) while this flag is
 // on, the result is undefined.
 //
-// #define SPDLOG_NO_THREAD_ID
+#define SPDLOG_NO_THREAD_ID
 ///////////////////////////////////////////////////////////////////////////////
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -57,7 +57,7 @@
 // Uncomment if logger name logging is not needed.
 // This will prevent spdlog from copying the logger name on each log call.
 //
-// #define SPDLOG_NO_NAME
+#define SPDLOG_NO_NAME
 ///////////////////////////////////////////////////////////////////////////////
 
 ///////////////////////////////////////////////////////////////////////////////
@@ -142,4 +142,4 @@
 // Defaults to __FUNCTION__ (should work on all compilers) if not defined.
 //
 // #define SPDLOG_FUNCTION __PRETTY_FUNCTION__
-///////////////////////////////////////////////////////////////////////////////
\ No newline at end of file
+///////////////////////////////////////////////////////////////////////////////
-- 
2.21.0

